lerobot/common/optim/optimizers.py

```diff
@@ -44,7 +45,7 @@ def default_choice_name(cls) -> str | None:
         return "adam"
 
     @abc.abstractmethod
-    def build(self) -> torch.optim.Optimizer:
+    def build(self) -> torch.optim.Optimizer | dict[str, torch.optim.Optimizer]:
```

Comment:
For SAC, and more generally for RL algorithms, we need multiple optimizers rather than just one. The format for multiple optimizers will be dict[str, torch.optim.Optimizer].